### DISCUSSION TITLE
I am a beginner and I would like to ask how to trigger node click events for custom nodes. Is there an example? I have looked at the document but couldn't find it

### DISCUSSION THREAD

**Discussion by yang1366788** [bcakmakoglu/vue-flow#1363] [Q&A]
I am a beginner and I would like to ask how to trigger node click events for custom nodes. Is there an example? I have looked at the document but couldn't find it

**Comment by bcakmakoglu** [comment]
You trigger click events by clicking a node 😄 

I'm guessing you want to know how to *listen* to a click event?
There's two ways to go about this

```html
<VueFlow @node-click="handleNodeClick">
```

or

```ts
const { onNodeClick } = useVueFlow()

onNodeClick((params) => {
	// ...
})
```

**Comment by R3stFu1** [comment]
How does this work with dblclick? ... I've been trying to use the @node-dblClick event handle, but it does not trigger the associated method.

**Reply by R3stFu1** [reply]
Never mind: Found it. 
The correct event handle is @node-double-click. Then it works.
